File: src/main/java/com/manicure/service/ItemCatService.java
package com.manicure.service;

import java.util.List;

import com.manicure.entity.TbItem;
import com.manicure.entity.TbItemCat;

public interface ItemCatService {
	
	/**
	 * 根据父 id 查询实体 
	 * @param id
	 * @return
	 */
	public List findByParentId(Long id);

	/**
	 * 根据id查询实体
	 * @param id
	 * @return
	 */
	public TbItemCat findOne(Long id);

	/**
	 * 查找所有分类
	 * @return
	 */
	public List<TbItemCat> findAll();

}

File: src/main/webapp/js/service/itemCatService.js
//服务层
app.service('itemCatService',function($http){
	    	
	//根据id获取商品信息
	this.findByParentId=function(id){
		return $http.get('../itemCat/findByParentId.do?id='+id);		
	}
	
	
	//根据id获取商品信息
	this.findOne=function(id){
		
		return $http.get('../itemCat/findOne.do?id='+id);		
	}
	
	//根据id获取商品信息
	this.findAll=function(){
		
		return $http.get('../itemCat/findAll.do');		
	}
	
	
	
});